conf/ocsci/rest_encryption.yaml

```diff
@@ -0,0 +1,6 @@
+---
+# This configuration file is used for enable REST encryption in the deployment
+# Valid only for OCS >= 4.6
+# https://issues.redhat.com/browse/KNIP-1264
+ENV_DATA:
+  rest_encryption: true
```

Comment:
I think we should have it disabled by default

Reply:
it will be disabled by default ;) Only when you pass this config file as described in MD file it will be enabled ;)

Reply:
ahh right!

Reply:
Will be fixed here:  https://github.com/red-hat-storage/ocs-ci/issues/2699